packages/gridfs/package.js

```diff
@@ -15,12 +15,12 @@ Package.onUse(function(api) {
   api.versionsFrom('1.0');
 
   api.use(['cfs:base-package@0.0.27', 'cfs:storage-adapter@0.1.1']);
-  api.addFiles('gridfs.server.js', 'server');
-  api.addFiles('gridfs.client.js', 'client');
+  api.addFiles('packages/gridfs/gridfs.server.js', 'server');
+  api.addFiles('packages/gridfs/gridfs.client.js', 'client');
 });
 
 Package.onTest(function(api) {
   api.use(['cfs:gridfs', 'test-helpers', 'tinytest'], 'server');
-  api.addFiles('tests/server-tests.js', 'server');
-  api.addFiles('tests/client-tests.js', 'client');
+  api.addFiles('packages/gridfs/tests/server-tests.js', 'server');
+  api.addFiles('packages/gridfs/tests/client-tests.js', 'client');
```

Comment:
@PEM-- not sure about this - this should be relative to the package folder when publishing. (if the case we would have to do major refactoring don't want to :)

gridfs might need need to published binaries (its the only binary package)